samples/net/sockets/echo_client/src/tcp.c

```diff
@@ -91,6 +102,41 @@ static int start_tcp_proto(struct data *data, struct sockaddr *addr,
 		return -errno;
 	}
 
+#if defined(CONFIG_SOCKS)
```

Comment:
Could we change this to `if (IS_ENABLED(CONFIG_SOCKS5) { `

Reply:
Done.